modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -603,6 +603,12 @@ public String toOperationId(String operationId) {
             operationId = "call_" + operationId;
         }
 
+        // operationId starts with a number
+        if (operationId.matches("^\\d.*")) {
+            LOGGER.warn(operationId + " (starting with a number) cannot be used as method name. Renamed to " + camelize(sanitizeName("call_" + operationId)));
+            operationId = "call_" + operationId;
```

Comment:
still not sanitizing here

Reply:
I am not sure to understand, this is just about adding the `call_` prefix. The value will be sanitized on line 612: `return camelize(sanitizeName(operationId));`

Reply:
oh my bad, didnt see that

Reply:
@jmini is correct. It was already sanitized a few lines above and that's the reason why I didn't call the function `sanitize` again when the operationId starts with a number.